sensu/plugins/check-vyatta-tunnels.py

```diff
@@ -59,11 +59,11 @@ def main():
         switch_on_criticality()
 
     for index, device in enumerate(device_list):
-        if " vti" in str(device):
```

Comment:
also the output shown in your comment is off by one (first shows 10,11 and second shows 9,10) however the result is the same.

iso.3.6.1.2.1.2.2.1.8.10 = INTEGER: 2
iso.3.6.1.2.1.2.2.1.8.11 = INTEGER: 1